config/prow/cluster/base/dind-docker-config_configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dind-docker-config
+  namespace: test-pods
+data:
+  daemon.json: |-
+    {
+      "cgroup-parent": "prowparent.slice"
```

Comment:
Does this cgroup need to exist beforehand?
Is this created explicitly somewhere?